9.tcga-classify/README.md

```diff
@@ -0,0 +1,125 @@
+# TCGA BioBombe Application
+
+**Gregory Way 2019**
+
+## Predicting cancer type and gene alteration status using compressed features
+
+In this module, we use the compressed latent features from 5 algorithms and across 28 different dimensions, to predict cancer type and gene alteration status in The Cancer Genome Atlas (TCGA) gene expression data.
+
+We predict cancer types and gene alterations using a logistic regression algorithm penalized with elastic net.
+We previously used a similar approach to predict Ras pathway activation in the same dataset ([Way et al. 2018](https://doi.org/10.1016/j.celrep.2018.03.046)).
+Rather than using raw gene expression features, we predict cancer types and mutations using compressed gene expression features.
+
+Our goal in this analysis was to determine the ability of the constructed features to learn signals representing cancer type and gene alterations.
+Our aim was to observe at which dimension and at what strength the signals appeared in the compression models.
+Furthermore, the compression algorithms aggregate groups of genes together that represent coordinated biological processes.
+Observing how these compressed features participated in the models (by model coefficients) we can make statements about the compressed features biologically.
+
+**Notes**
+
+* Samples with a hypermutater phenotype were removed from the dataset (greater than 5 standard deviations over the mean of log10 mutation rate)
+* All models are adjusted for log10 mutation rate
+* Gene alteration predictions only:
+  * These models are also adjusted by cancer type
+  * To regulate balanced class sizes, only cancer types with greater than 15% and more than 5 samples in either positive or negative gene alteration class were used to train models
+  * Gene alteration was defined as any non-silent mutation in the gene of interest, or a copy number amplification for an oncogene, or copy number deletion for a tumor suppressor gene.
+
+### Predicting cancer types
+
+TCGA has profiled 33 different cancer-types.
+We constructed supervised machine learning tasks to distinguish each cancer-type from all others.
+
+### Predicting gene alterations
+
+We selected the [top 50 most mutated genes](https://github.com/greenelab/BioBombe/blob/master/9.tcga-classify/top-50-pancanatlas-mutations.ipynb) in TCGA to predict.
```

Comment:
Might be too late to change this.  I assume 50 genes was an arbitrary cutoff, I wonder if you tried looking at the distribution to help determine the threshold?

Reply:
yeah, 50 was arbitrary. I didn't look at the distribution myself, but this is well studied in this dataset. I think it would be really cool to predict all mutations and plot signal we observe. Probably beyond scope of this analysis though